app/src/main/shared/components/OneTimeButton/OneTimeButton.tsx

```diff
@@ -31,7 +32,11 @@ export class OneTimeButton extends React.Component<OneTimeButtonProps, any> {
     }
 
     refreshToken() {
-        setTimeout(this.props.refreshToken);
+        if (typeof this.props.refreshToken === "function") {
+            setTimeout(() => {
+                this.props.refreshToken();
+            });
```

Comment:
Why not just:
```
if (typeof this.props.refreshToken === "function") {
    setTimeout(this.props.refreshToken);
}
```

Reply:
left from testing something, yeah, will put it back, as it was before